aiida/Dockerfile

```diff
@@ -1,38 +1,42 @@
 # For finding latest versions of the base image see
 # https://github.com/SwissDataScienceCenter/renkulab-docker
-FROM renku/renkulab-py:3.10-0.21.0
+FROM renku/renkulab-py:3.10-0.22.0
+
+# Effectively disable RabbitMQ `consumer_timeout`
+ENV RABBITMQ_SERVER_ADDITIONAL_ERL_ARGS="-rabbit consumer_timeout undefined"
```

Comment:
One thing that just occurred to me - it could be that you need to add this to the `~/.bashrc` rather than the container environment since RMQ is actually started from a subshell in the entrypoint?

Reply:
I just tested it with `rabbitmqctl eval 'application:get_env(rabbit, consumer_timeout).'` as suggested in the [Stackoverflow post you linked](https://stackoverflow.com/questions/70957962/rabbitmq-consumer-timeout-behavior-not-working-as-expected/73928874#73928874).

The behavior seems to be OK:

![image](https://github.com/SwissDataScienceCenter/contributed-project-templates/assets/41700727/8de06972-d6d3-4026-bac2-c8e76c2301fa)

Haven't tested it by actually running AiiDA yet, though.